lib/src/common/disposable.dart

```diff
@@ -436,6 +463,7 @@ class Disposable implements _Disposable, DisposableManagerV3, LeakFlagger {
     _internalDisposables.add(disposable);
   }
 
+  @deprecated
```

Comment:
Are there any instances where consumers are passing stream subscription references between classes?

I know there are some instances where stream subscription lifetimes are less that that of their disposable parent. Consumers often use lists of subscriptions to batch them for cancellation. I would be interested if that behavior would extend across class instances.